### ISSUE TITLE
Add property "donor_reports" to the Project entity

### ISSUE THREAD

**Issue by cezar77** [ilri/livegene#5]
We have to add property **donor_reports** to the entity `Project`. It should contain any links to the donor reports. It should be an optional property and it can hold many links.

I'd prefer to store all links as an array into one column.  
We have to explore the new possibilities of MySQL 5.7 (https://dev.mysql.com/doc/refman/5.7/en/json.html).

**Comment by Mirieri** [comment]
The correct naming convention is **donarReports**

**Comment by cezar77** [comment]
@Mirieri you're right, just there is a typo, it should be **donorReports**.

**Comment by cezar77** [comment]
I have created a branch "donorreports" and added the property **donorReports** to the `Project` entity. It is a `simple_array` and is stored in the database as nullable LONGTEXT. The values in the field are separated by comma. Doctrine takes care for serialization and deserialization.

I have added the property to the `ProjectAdmin` too. It uses the `CollectionType` widget from Symfony. However it requires some tuning up in the SHOW action.  
We might possibly need a custom template for displaying the links.
